doc/classes/Basis.xml

```diff
@@ -114,7 +115,18 @@
 			<argument index="0" name="scale" type="Vector3">
 			</argument>
 			<description>
-				Introduce an additional scaling specified by the given 3D scaling factor. Only relevant when the matrix is being used as a part of [Transform].
+				Introduce an additional scaling specified by the given 3D scaling factor.
+			</description>
+		</method>
+		<method name="slerp">
```

Comment:
@tagcup FYI, bindings were missing for `Basis.slerp`, but since you wrote the doc for it I assumed they were wanted and added them in 4c5dd51eb.